src/api/error/ErrorKinds.ts

```diff
@@ -15,4 +15,5 @@ export const ErrorKinds = {
   APP_LOGS_SSE_ERROR: 'LogsSSEError',
   APP_LOGS_PARSE_ERROR: 'LogsParseError',
   STICKY_HOST_ERROR: 'StickyHostError',
+  FLOW_ISSUE_ERROR: 'FlowIssue',
```

Comment:
What do you think about maintaining the pattern of the other constants? 
```suggestion
  FLOW_ISSUE_ERROR: 'FlowIssueError',
```

Reply:
I think that this a good Idea, but we discussed this already with Molgado and they decided to use without `Error`